front-end-node/NodeInspectorOverrides.js

```diff
@@ -134,3 +150,47 @@ function getAllUiSourceCodes() {
     return p.name();
   }
 }
+
+var notifications = [];
+
+function processDebuggerPaused(event) {
+  var reason = event.data.reason;
+  var lastFrame = event.data.callFrames[0];
+  var title, message;
+  switch (reason) {
+    case 'exception':
+      title = 'Exception caught';
+      if (lastFrame) {
+        message = 'Exception caught in module ' + (lastFrame._script.sourceURL||('(' + lastFrame._location.scriptId + ')')) +
+          ' at line ' + lastFrame._location.lineNumber + ' column ' + lastFrame._location.columnNumber +
+          '.\n\nClick here to inspect.';
+      }
+      break;
+    case 'other':
+    default:
+      title = 'Debugger paused';
+      if (lastFrame) {
+        message = 'Breakpoint caught in module ' + (lastFrame._script.sourceURL||('(' + lastFrame._location.scriptId + ')')) +
+          ' at line ' + lastFrame._location.lineNumber + ' column ' + lastFrame._location.columnNumber +
+          '.\n\nClick here to inspect.';
+      }
+      break;  
+  }
+  var notification = new Notification(title, {
+    body: message||'Click here to inspect.',
+    icon: '/favicon.ico'
+  });
+  notifications.push(notification);
+  notification.addEventListener('click', function(event) {
+    this.close();
+    window.focus();
+  });
+  ['click','focus'].forEach(function(attachTo) {
```

Comment:
You're right, I move it within the _enableDebuggerUINotifications function for window events